### PR TITLE
Add django 2.2 compatibility

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Add django 2.2 compatibility

### CODE REVIEW

File: eox_core/settings/test.py [hunk 1]
```diff
@@ -38,7 +38,11 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
 plugin_settings(SETTINGS)
 vars().update(SETTINGS.__dict__)
 
-INSTALLED_APPS += ('eox_core', )
+try:
+    import edx_when  # pylint: disable=unused-import
+    INSTALLED_APPS += ('eox_core', 'edx_when.apps.EdxWhenConfig')
```
Comment (morenol) [inline eox_core/settings/test.py:43]: Why is this needed?

Comment (andrey-canon) [reply eox_core/settings/test.py:43]: @morenol edx-proctoring(2.4.0) adds this requirement and there is a runtime error if this is not included in the installed apps

**Review by morenol — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]
